lib/v1_alpha_fetcher.rb

```diff
@@ -0,0 +1,45 @@
+class V1AlphaFetcher
+  def self.fetch(url)
```

Comment:
was there any thought as to why we've made this static instead of an instance method?